addon/components/time-series-chart.js

```diff
@@ -58,6 +58,12 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
   // Bar left offset, as fraction of width of bar
   barLeftOffset: 0.0,
 
+  // Force X-Axis labels to print vertically
+  xAxisVertLabels: false,
+
+  // Enable smart turning if feasible - TRUMPED by xAxisVertLabels
+  xAxisSmartLabels: false,
+
```

Comment:
Actually I think we're going to remove the smart turning feature, so this will be far more straightforward.